client/my-sites/plans/plan-overview/plan-status/plan-status-progress/style.scss

```diff
@@ -0,0 +1,28 @@
+.plan-status-progress {
+	display: flex;
+	flex-wrap: wrap;
+	margin-left: -24px;
+
+	&.is-expiring {
+		.plan-status-progress__time-until-expiry {
+			color: $alert-red;
+		}
+
+		.progress-bar__progress {
+			background-color: $alert-red;
+		}
+	}
+}
+
+.plan-status-progress__progress-bar,
+.plan-status-progress__time-until-expiry {
+	margin-left: 24px;
+}
+
+.plan-status-progress__progress-bar {
```

Comment:
Not a big deal but we could rename that class into `plan-status-progress__bar` so we don't repeat `progress` here.